app/models/concerns/blacklight/marc/document_export.rb

```diff
@@ -572,6 +573,7 @@ def name_reverse(name)
     name = clean_end_punctuation(name)
     return name unless name =~ /,/
     temp_name = name.split(", ")
+    return name if temp_name.empty?
```

Comment:
Does this handle some case other than `name = ", "`?  If so, can we just look for that on line 574?  Otherwise, would you mind adding some comments to explain what we're looking for?